Readme.md

```diff
@@ -1,84 +1,195 @@
+</div>
 
-- [ ] users rsvp events, and a total is displayed
-- [ ] discord bot that will message users for events they are rsvp'd in?
-- [ ] events can have channels tagged that match the server channel emojis' daily view. Clicking on the day can bring up a day planner view, just like the google calendar
-- [ ] users can add an event to their calendar through a click
-- [ ] users can filter events based on pre-determined tags (beginner-friendly, networking, group learning)
-- [ ] users can share events on Twitter
-- [ ] new event notification dots(easy mode might be to just highlight new events on each reload)
 
-## The Penultimate Goal
+<br>
 
-- [ ] Discord guilds can create their own together calendar
+## Help us improve Together!
 
-## thinking ahead,
+<em>Together</em> is a community-driven open source project backed by 100devs students. Whether you are helping us fixing bugs, proposing new feature, improving our documentation or spreading the word - <strong>All contributions are welcome.</strong>
 
-keep these in mind to prevent the project from being "caged."
-can integrate with slack or Discord and work on other servers besides 100devs
+<br>
 
-Resources:
-https://codepen.io/robstinson/pen/BaKOZry
+If you're interested in contributing, you may:
+* read our [Contributing guide](.github/CONTRIBUTING.md);
+* Visit the [Issues tab](https://github.com/Caleb-Cohen/Together/issues) to request an issue or to open a new issue;
+* You can also convert a "draft todo" to an issue on the [MVP Tasks Project Board.](https://github.com/users/Caleb-Cohen/projects/1/views/1)
 
----
+<p align="right">(<a href="#top">back to top</a>)</p>
 
-# Things to add
 
-- Create a `.env` file in the config folder and add the following as `key = value.`
-  - PORT = 2121 (Do not change)
-  - DB_STRING = `your database URI`
-  - CLOUD_NAME = `your Cloudinary cloud name`
-  - API_KEY = `your Cloudinary API key`
-  - API_SECRET = `your Cloudinary API secret`
-  - DISCORD_CLIENT_ID = `discord client ID`
-  - DISCORD_CLIENT_SECRET = `discord client secret`
-  - OAUTH_REDIRECT_URL = http://localhost:3000/
 
----
+<div align='center'>
+  <img src="docs/images/divider.png">
+</div>
 
-# Install & Run
+# 📌 Acknowledgements
 
-- ensure `.env` file is located in `server/config` file.
-- `npm install` in the root folder. installs the required dependencies.
-- `npm run build` compiles the build folder. Lint must have no errors for build file command to complete. 
-- `npm run dev-concurrent` in the root folder.
+<div align="center">
+<table>
+  <tr>
+    <td valign="center">
+     <h3>Resources</h3>
+        · Tailwind Calendar by <a href="https://codepen.io/robstinson/pen/BaKOZry">Rob Stinson</a> @ Codepen
+        <br>
+        · 'Cool Kids' illustrations by <a href="https://blush.design/artists/iDxEJwP2Ha4IrbT6bF88/irene-falgueras">Irene Falgueras</a>
+    </td>
+    <td valign="center" style="width:10%"><img height="120px" src="docs/images/mug.png" alt="Mug"/></td>
+  </tr>
```

Comment:
A bit hard to type up the exact change in the editor, but can you add yourself to this?

"Readme Designed by [Praxeds/Ana](https://github.com/praxeds/)"